packages/components/dashboard/resource-providers-card/resource-providers-card.tsx

```diff
@@ -11,105 +11,63 @@ import {
   GalleryItem,
   Skeleton,
 } from '@patternfly/react-core';
-import { MCG_MS_PROMETHEUS_URL } from '../../../constants';
-import { getMetric } from '../../../utils';
```

Comment:
do we need `getMetric` in the utils? if not we can remove it from there also as I added it during the resource provider card

Reply:
it will be useful soon... it is a small function, we can let it be there...

Reply:
cool